app/javascript/pulfalight/pulfalight_loader.es6

```diff
@@ -45,6 +46,17 @@ export default class {
     toc.build()
   }
 
+  setup_toc_collapse() {
+    const mediaQuery = window.matchMedia('(max-width: 768px)')
+    function handle_mobile_change(e) {
```

Comment:
I'm not sure how I feel about having this function defined inside the other function. Not a good sense of javascript style conventions here, but my initial reaction is that I'd like to see it defined outside.